ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -14,6 +14,8 @@ parameters:
   required: true
 - name: IMAGE_AZURE_CONTROLLERS
   required: true
+- name: IMAGE_METRICSBRIDGE
+  required: true
```

Comment:
Why are these required? Shouldn't we default to a value if none is passed?

Reply:
Required because image is built with each PR, and we want to test with it. I better make it mandatory than miss some bug.

Reply:
How is this going to affect us running the template locally?

Reply:
It should not. otherwise, we would already see "the issue" with `IMAGE_SYNC` and `IMAGE_AZURE_CONTROLLERS` as they use the same pattern.